pages/index.html

```diff
@@ -1,12 +1,41 @@
+<!--must include your name somewhere-->
 <!DOCTYPE html>
```

Comment:
It is a convention to organize files such that the `index.html` is at the project root directory, and the other pages are organized in a directory such as `pages`